packages/mui-joy/src/styles/CssVarsProvider.tsx

```diff
@@ -25,22 +25,17 @@ const {
   },
 });
 
-let warnedInitScriptOnce = false;
+/**
+ * @deprecated use `InitColorSchemeScript` instead
+ *
+ * ```diff
+ * - import { getInitColorSchemeScript } from '@mui/joy/styles';
+ * + import InitColorSchemeScript from '@mui/joy/InitColorSchemeScript';
+ *
+ * - getInitColorSchemeScript();
+ * + <InitColorSchemeScript />;
+ * ```
+ */
+export const getInitColorSchemeScript = getInitColorSchemeScriptSystem;
```

Comment:
I've added redeclaration to improve the jsdoc, because it doesn't work well when added to reexport:

| Before | After |
| ------- | ----- |
| <img width="734" alt="Screenshot 2024-07-05 at 11 53 58" src="https://github.com/mui/material-ui/assets/13808724/09f958e4-6f06-453c-8e53-83206f42bf62"> | <img width="738" alt="Screenshot 2024-07-05 at 11 53 26" src="https://github.com/mui/material-ui/assets/13808724/05413a2a-ee91-48e0-8139-9466f7c0c292"> |